CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ For migration information, you can always have a look at https://liip-drifter.re
 
 ### Fixed
 
+- Adapt postgis role to version 2
```

Comment:
This should be in the "Changed" section.

Reply:
Moved to Unreleased, Changed.